### ISSUE TITLE
I would like to know which software was used to make overlays

### ISSUE THREAD

**Issue by TejasLamba2006** [KevinTrinh1227/Hycord-Bot#7]

**Comment by KevinTrinh1227** [comment]
I'm unsure if I understand your question, but here is the list of tools I used to make the images... 

I used the Pillow Library (aka PIL) -> https://pypi.org/project/Pillow/
And photo editing I used: https://pixlr.com/x/ where I then created overlays, and editing the backgrounds and assets.

And essentially how the commands work is that they will layer each item/asset over each other and then draw on members stats or whatever context over the image then send it in the channel the command was issued in. Of course, all using the PIL library. 

Example:
I get the top 30 guild members names and how many points they scored, then sort them, then layer each text over the overlay. and this is the output image:
![image](https://github.com/KevinTrinh1227/Hycord-Bot/assets/48145892/c32277a5-2361-4f71-96f6-d2ccebbf8154)
